ushahidi/pwa-prototype-reporting
# offline support updated

This pull request 
- determine when the user is offline
- If offline, store the data in localStorage
- When the user comes online, read the data from localStorage and make POST requests for each entry in the Array
- remove the items from localStorage after you send them

How to Test:
1. Check if the data is posted or not when the user is online in the console.
2. Now, when the user is offline. Check the local storage, if the failed submission data is stored or not.
3. When the user comes online again check if the key is removed or not when a post request is made in the local storage.
